source/catalog.html

```diff
@@ -0,0 +1,302 @@
+    <div class="add-offers__present">
+      <img class="present__image" src="img/cat-in-bag.png" width="279" height="289" alt="Переноска для кота">
+    </div>
+  </section>
+  <section class="cooperation">
+    <div class="cooperation__wrapper">
+      <h2 class="cooperation__title">Приглашаем к сотрудничеству дилеров!</h2>
+      <address class="cooperation__addres">
+        ул. Большая Конюшенная, д. 19/8 Санкт-Петербург
+      </address>
+      <div class="cooperation__map">
+
+      </div>
+    </div>
+  </section>
+</main>
+<footer class="footer">
+  <svg class="logo-title" xmlns="http://www.w3.org/2000/svg" width="101" height="19" fill="none"><g fill="#111" clip-path="url(#a)"><path d="M1.264 17.346A6.855 6.855 0 0 1 0 13.042v-6.38a6.94 6.94 0 0 1 1.185-4.38c.87-1 2.213-1.461 4.189-1.461s3.24.461 4.03 1.306a5.415 5.415 0 0 1 1.186 3.767V7.43H6.718V5.663a3.687 3.687 0 0 0-.238-1.537 1.12 1.12 0 0 0-.456-.472 1.164 1.164 0 0 0-.65-.143c-.553 0-.948.154-1.106.538A4.838 4.838 0 0 0 4.03 5.74v8.224a3.107 3.107 0 0 0 .316 1.614c.158.385.553.538 1.106.538s.87-.153 1.027-.538c.18-.52.26-1.066.238-1.614V12.12h3.872v1.538A5.767 5.767 0 0 1 9.405 17.5c-.79.922-2.134 1.383-4.031 1.383-1.897-.077-3.32-.538-4.11-1.537ZM15.253 1.053h4.504l3.32 17.524H19.44l-.632-3.69h-2.608l-.632 3.69h-3.715l3.399-17.524Zm3.24 11.529-.948-6.84-.949 6.84h1.897ZM25.843 3.973h-2.687v-2.92h9.404v2.844h-2.687v14.68h-4.03V3.973ZM39.752 1.053h6.955v1.844h-4.505v5.765h3.635v1.768h-3.635v6.456h4.505v1.768h-7.034V1.053h.079ZM48.761 1.053H50.5l5.137 11.606V1.053h2.055v17.524h-1.58l-5.217-11.99v11.99h-2.134V1.053ZM60.063 1.053h6.954v1.844h-4.504v5.765h3.635v1.768h-3.635v6.456h4.504v1.768h-6.954V1.053ZM69.072 1.053h3.635a6.455 6.455 0 0 1 4.03 1.076c.87.692 1.265 1.921 1.265 3.536 0 2.228-.71 3.612-2.212 4.073l2.528 8.762h-2.45l-2.37-8.224h-1.976V18.5h-2.529V1.053h.079Zm3.477 7.532a3.526 3.526 0 0 0 2.292-.615c.474-.384.711-1.153.711-2.305a4.838 4.838 0 0 0-.237-1.691 1.973 1.973 0 0 0-.869-.923 3.693 3.693 0 0 0-1.739-.307h-1.106v5.764h.948v.077ZM81.163 17.27a7.602 7.602 0 0 1-1.106-4.458V6.971c-.02-1.114.114-2.225.395-3.305a3.913 3.913 0 0 1 1.502-1.999 5.277 5.277 0 0 1 2.924-.691c1.66 0 2.845.384 3.556 1.23a5.763 5.763 0 0 1 1.107 3.843v.614H87.17v-.538c.003-.643-.05-1.286-.158-1.921a1.532 1.532 0 0 0-.633-1 2.495 2.495 0 0 0-1.501-.384 2.418 2.418 0 0 0-1.58.462c-.187.152-.338.34-.447.551a1.7 1.7 0 0 0-.186.678c-.104.687-.157 1.381-.158 2.076v6.533a6.807 6.807 0 0 0 .474 2.92c.316.615.949.846 1.897.846a1.96 1.96 0 0 0 1.092-.182c.337-.164.617-.422.805-.74a6.898 6.898 0 0 0 .553-3.075v-1.23h-2.292V9.892h4.584v8.685h-1.58l-.238-1.999a3.141 3.141 0 0 1-1.226 1.657 3.29 3.29 0 0 1-2.014.573 4.049 4.049 0 0 1-1.899-.352 3.932 3.932 0 0 1-1.5-1.186ZM94.678 12.43l-3.873-11.3h2.371l2.766 8.455 2.687-8.454H101l-3.873 11.298v6.15h-2.37v-6.15h-.08Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 .82h101v17.986H0z"/></clipPath></defs></svg>
+  <ul class="footer__social-media__list">
```

Comment:
А содержимое сделай элементами `social`.
```suggestion
  <ul class="footer__social social">
```